packages/fxa-settings/src/components/PageAvatar/buttons.tsx

```diff
@@ -171,19 +171,25 @@ export const ConfirmBtns = ({
     <div className="mt-4 flex items-center justify-center">
       <Localized id="avatar-page-close-button">
         <button
-          className="cta-neutral mx-2 px-10"
+          className="cta-neutral mx-2"
           onClick={() => navigate(HomePath, { replace: true })}
           data-testid="close-button"
+          style={{
+            width: '25%',
+          }}
```

Comment:
So while this works we do like to avoid inline styles wherever possible. It also gives us a funky layout at smaller screens:

![Screen Shot 2021-04-06 at 10 21 42 AM](https://user-images.githubusercontent.com/6392049/113726041-e3d25900-96c1-11eb-82ac-57e637cb8670.png)

Since these buttons are flex children they can be expanded and squeezed without breaking, so what if we allow each button to grow as wide as it can to a maximum width, something like `w-full max-w-48`. I used `max-w-48` because it appears to be the smallest max-width [we have configured](https://github.com/mozilla/fxa/blob/main/packages/fxa-react/configs/tailwind.js#L55:L55), but it's still a little too wide. We could add `max-w-8` (2rem) for a more reasonable max-width.

Reply:
Yeah, your solution is much prettier than mine. I tried to make styling flexible using percents but I need to get used to the Tailwind classes and functionalities... Thank you!
I updated the PR